backend/epub/epub-document.c

```diff
@@ -643,7 +643,7 @@ check_mime_type(const gchar* uri,GError** error)
             mimeFromFile = ev_file_get_mime_type (uri, TRUE, &err);
             if (mimeFromFile)
             {
-                for (i = 0; i < g_strv_length (mimetypes); i++) {
+                for (i = 0; i < g_strv_length ((gchar**) mimetypes); i++) {
```

Comment:
If you choose this solution, let's apply it here as well
```suggestion
                for (i = 0; mimetypes[i]; i++) {
```